src/libsaml.ts

```diff
@@ -457,44 +456,50 @@ const libSaml = () => {
 
         sig.loadSignature(signatureNode);
 
-        doc.removeChild(signatureNode);
-
         verified = sig.checkSignature(doc.toString());
 
         // immediately throw error when any one of the signature is failed to get verified
         if (!verified) {
-          throw new Error('ERR_FAILED_TO_VERIFY_SIGNATURE');
+          continue;
+          // throw new Error('ERR_FAILED_TO_VERIFY_SIGNATURE');
         }
```

Comment:
Confusingly, `checkSignature` [now sometimes returns `false`](https://github.com/node-saml/xml-crypto/blob/9b91edf61cbc83853d824933be0dafa1f6114de4/src/signed-xml.ts#L336-L339), but usually throws an error when unable to validate the signature. 
When I upgraded from 2.9.1 to 2.10.0, I had tests start to fail because some error handling was looking for `ERR_FAILED_TO_VERIFY_SIGNATURE` (to provide custom messaging). Is the expectation now that the xml-crypto errors are free to bubble up? or should `checkSignature` be wrapped in a try/catch and the xml-crypto error passed as a `cause`?